source-mysql/README.md

```diff
@@ -0,0 +1,119 @@
+Flow MySQL Source Connector
```

Comment:
This is an exceptionally well written README, as is the one for source-postgres! 👌

Reply:
Thanks!